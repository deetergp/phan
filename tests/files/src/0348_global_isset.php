<?php declare(strict_types=1); // presence of strict_types affects context merging behavior

if (isset($global348)) {
    echo "Var $global348 is set";
}
var_dump($global348);  // should warn

if (!isset($globalB348)) {
    $globalB348 = true;
}
var_dump($globalB348);  // should not warn
var_dump($globalC348);  // should warn
