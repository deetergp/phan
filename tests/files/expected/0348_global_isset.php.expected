%s:6 PhanUndeclaredVariable Variable $global348 is undeclared
%s:12 PhanUndeclaredVariable Variable $globalC348 is undeclared
